users/guilisten.rst

```diff
@@ -0,0 +1,52 @@
+.. _gui-listen:
+
+The GUI Listen Address
+======================
+
+The GUI (and, together with it, the REST API) listens on a single TCP port
+for HTTP and HTTPS connections. By default this address is ``127.0.0.1:8443``.
```

Comment:
Should be port 8384 as written below. Is also incorrect in the help string for the `-gui-address` option.